app/api/submission.py

```diff
@@ -1,104 +1,179 @@
-    """Takes a Submission Object and calls the Google Vision API to text annotate
-    the passed s3 link, then passes those concatenated transcriptions to the SquadScore
-    method, returns:
-
-    Arguments:
-    ---
-    `sub`: Submission - Submission object **see `help(Submission)` for more info**
-    Returns:
-    ---
-    ```
-    {"SubmissionID": int, "IsFlagged": boolean,"LowConfidence": boolean, "Complexity": int}
-    ```
-    """
-    transcriptions = ""
-    confidence_flags = []
-    # unpack links for files in submission object
-    for page_num in sub.Pages:
-        # re-init the sha algorithm every file that is processed
-        hash = sha512()
+    '''
+    This is a test endpoint that is a duplicate of the illustration submission endpoint 
+    that allows the web dev team to test the endpoints.
+    '''
         # fetch file from s3 bucket
-        r = get(sub.Pages[page_num]["URL"])
-        # update the hash with the file's content
-        hash.update(r.content)
-        try:
-            # assert that the hash is the same as the one passed with the file
-            # link
-            assert hash.hexdigest() == sub.Pages[page_num]["Checksum"]
-        except AssertionError:
-            # return some useful information about the error including what
-            # caused it and the file affected
-            return JSONResponse(
-                status_code=422,
-                content={"ERROR": "BAD CHECKSUM", "file": sub.Pages[page_num]},
-            )
-        # unpack response from GoogleAPI
-        conf_flag, flagged, trans = await vision.transcribe(r.content)
-        # concat transcriptions togeather
-        transcriptions += trans + "\n"
-        # add page to list of confidence flags
-        confidence_flags.append(conf_flag)
-    # score the transcription using SquadScore algorithm
-    score = await squad_score(transcriptions, scaler)
-
-    # return the complexity score to the web team with the SubmissionID
+    r = get(sub.URL)
+    # initalize the sha hashing algorithm
+    hash = sha512()
+    # update the SHA with the file contents
+    hash.update(r.content)
+    # assert that the computed hash and the passed hash are the same
+    try:
+        assert hash.hexdigest() == sub.Checksum
+    except AssertionError:
+        # return bad hash error with the status_code to an ill-formed request
+        return JSONResponse(status_code=422, content={"ERROR": "BAD CHECKSUM"})
+    # Make prediction with the model
+    prediction =  await model.predict(r.content)
+    # respond with the output dictionary that is returned from
+    # model1.predict() method
     return JSONResponse(
-        status_code=200,
+        status_code=200, 
         content={
             "SubmissionID": sub.SubmissionID,
-            "IsFlagged": flagged,
-            "LowConfidence": True in confidence_flags,
-            "Complexity": score,
-        },
+            "Complexity": prediction
+        }
     )
 
 
+
```

Comment:
lines 54-108 
during product review, we decided to leave this connected to the GV API and just get the credentials until MVP has been met on the model
please add this code back
thank you